plugins/zendesk/api_token.go

```diff
@@ -0,0 +1,71 @@
+package zendesk
+
+import (
+	"github.com/1Password/shell-plugins/sdk"
+	"github.com/1Password/shell-plugins/sdk/importer"
+	"github.com/1Password/shell-plugins/sdk/provision"
+	"github.com/1Password/shell-plugins/sdk/schema"
+	"github.com/1Password/shell-plugins/sdk/schema/credname"
+	"github.com/1Password/shell-plugins/sdk/schema/fieldname"
+)
+
+func APIToken() schema.CredentialType {
+	return schema.CredentialType{
+		Name:    credname.APIToken,
```

Comment:
The three credentials that are required for authentication are `Subdomain`, `Email`, and `API token`. I'm unsure about the `credname` as well as `fieldname` values I used, please let me know if I should change them to make it clearer which credentials the plugin needs.

Reply:
Personally, I have defaulted to `Credentials` as the credential name when I am unclear on the best name, but in this case, defining `LoginDetails` as a cred name and using it sounds good to me. I don't see defining new cred names as harmful. Even if, in the future, we need to define alternative credential names, something like defining [alternative field names](https://github.com/1Password/shell-plugins/pull/188) might be possible.

Reply:
I think `credname.APIToken` is fine as is. That's the actual thing you're creating/deleting and that's how Zendesk refers to it.

Reply:
I'd say we should refer to them as Zendesk does, since that's what would make most sense from the user-perspective, when creating a new plugin item.

So if they use `Subdomain` instead of `OrgURL` and `Email` instead of `Username`, let's go with that.

Reply:
@hculea That makes sense, just made a change to reflect that!